management/virtualmachine/resourceextensions_test.go

```diff
@@ -0,0 +1,38 @@
+package virtualmachine
+
+import (
+	"flag"
+	"testing"
+
+	"github.com/MSOpenTech/azure-sdk-for-go/management/testutils"
+)
+
+func init() {
+	flag.BoolVar(&extraVerbose, "resourceextensions.verbose", false, "List all resource extensions")

```

Comment:
But it is very documented... that's exactly what the fourth parameter of [BoolVar](http://golang.org/pkg/flag/#BoolVar) does package does. Now merge it already if you're happy with 4440be0! :smile: